common/src/main/java/com/microsoft/identity/common/nativeauth/internal/controllers/BaseNativeAuthController.kt

```diff
@@ -69,15 +69,6 @@ abstract class BaseNativeAuthController : BaseController() {
         throw ClientException("acquireToken() not supported in NativeAuthController")
     }
 
```

Comment:
could you update your PRs to include an overview of the changes?

Reply:
Done